lib/Form/DisplayFormInWPContent.php

```diff
@@ -156,7 +156,6 @@ private function getContentBellow(FormEntity $form, string $displayType): string
     $templateData['delay'] = $formSettings['form_placement'][$displayType]['delay'] ?? 0;
     $templateData['position'] = $formSettings['form_placement'][$displayType]['position'] ?? '';
     $templateData['animation'] = $formSettings['form_placement'][$displayType]['animation'] ?? '';
-    $templateData['backgroundColor'] = $formSettings['backgroundColor'] ?? '';
```

Comment:
I think we need to remove this from [the preview page](https://github.com/mailpoet/mailpoet/blob/master/lib/Form/PreviewPage.php#L100).